docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -53,4 +54,4 @@ See https://github.com/che-incubator/devworkspace-api/issues/4[corresponding iss
 * Renaming dockerimage component type. See  https://github.com/che-incubator/devworkspace-api/issues/8[corresponding issue].
 * Specify sources path for containers. See  https://github.com/che-incubator/devworkspace-api/issues/17[corresponding issue].
 * Specify size of volume for component. See https://github.com/che-incubator/devworkspace-api/issues/14[corresponding issue].
-* Containers endpoints (routes/ingresses). See https://github.com/che-incubator/devworkspace-api/issues/33[corresponding issue].
\ No newline at end of file
+* Containers endpoints (routes/ingresses). See https://github.com/devfile/api/issues/27[corresponding issue].
```

Comment:
What is the difference between links mentioned between Procedure and Additional Resource? 
1. I see they have the same heading
2. Not all have the right links - some redirect to wrong topics

Reply:
I had the same concerns, Maysun. And we plan to open up a separate issue for these concerns. For instance, in PR #39, while doing a language review, we noticed the structure needs changing as well (as you pointed out). We'll file a separate issue for these structural concerns so we better adhere to procedure documentation.

Reply:
do you want to update the links with this PR or with a new PR because some of them are wrong

Reply:
Hey Maysun, I've updated the links in this PR and in #39.  

The proper structure for links is, `https://github.com/devfile/api/issues/issue number` correct? Looking at this PR here, I don't see links that don't match that structure.

Reply:
cool, I did click thru all the links and they match the subject. But i would honestly like more content on this doc to actually tell how one can migrate components from devfile v1 to devfile v2. 

If you want to do that with some other PR or even 39, I can lgtm this.

Reply:
@maysunfaisal, yes, let's address that concern with links in #39 or possibly another PR. We might want to focus exclusively on style guide issues for #39 and make a separate PR for more content creation, like you suggested here with adding content to explain the links.

Reply:
cool, marking as resolved!